docs/develop-resources/guides/5-releases.md

```diff
@@ -1,14 +1,10 @@
 # 5. Release
 
-## 5.1 Turn on your Direct controller (TF/DCL Beta Only)
+## 5.1 Make Direct controller the default (for migration from TF/DCL)
 
-### For TF-based Beta resource
+* Remove the `cnrm.cloud.google.com/dcl2crd: "true"` or `cnrm.cloud.google.com/tf2crd: "true"` go tag from the CRD struct, and run `dev/tasks/generate-crds` to use Direct as the permanent controller.
```

Comment:
I like the wording changes, wan we also keep the example? Otherwise it could be hard for contributors to figure out what the `CRD struct` or `API` refers to.